js/renovation/ui/scroll_view/scrollbar.tsx

```diff
@@ -438,7 +402,7 @@ export class Scrollbar extends JSXComponent<ScrollbarPropsType>() {
   }
 
   get inRange(): boolean {
```

Comment:
It's difficult to understand how the 'inRange' works by its name:

- Component name is 'ScrollBar'
- Function name is 'inRange'
- Checked variable is scrollLocation
- Checked min/max variables are min/maxOffset